src/ml-avatar.js

```diff
@@ -0,0 +1,32 @@
+import { Avatar } from 'antd'
+import React from 'react'
+import PropTypes from 'prop-types'
+import { UserOutlined } from '@ant-design/icons'
+
+/**
+ * A default icon is shown unless children or `src` are present.
+ */
+const MLAvatar = (props) => {
+  return (
+    <Avatar icon={props.children || props.src ? null : <UserOutlined />} {...props}>
```

Comment:
It's not obvious to me what the order of operations is on `props.children || props.src ? null : <UserOutlined />`; could you add some parens to make it clear?